File: c#/ProjetoDM102/PessoaFisica.cs
using System;
namespace ProjetoDM102
{
    public class PessoaFisica : Cliente
    {
        public String cpf { get; }
		private String identidade;
		private String tipoIdentidade;

        public PessoaFisica(string nome,string endereco, string telefone,
                            string cpf, string identidade, string tipoIdentidade) : base ( nome,  endereco,  telefone)
        {
            this.cpf = cpf;
            this.identidade = identidade;
            this.tipoIdentidade = tipoIdentidade;
        }

		public override string ToString()
		{
		    return "[Pessoa Física] " + base.ToString() + " CPF: " + cpf + ", Identidade: " + identidade + ", Tipo de identidade: " + tipoIdentidade;
		}
    }
}

File: java/src/br/inatel/pos/mobile/dm102/projeto/model/PessoaFisica.java
package br.inatel.pos.mobile.dm102.projeto.model;

public class PessoaFisica extends Cliente {
	
	private String cpf;
	private String identidade;
	private String tipoIdentidade;
	
	public PessoaFisica(Cliente cliente, String cpf, String identidade, String tipoIdentidade) {
		super(cliente.getNome(), cliente.getEndereco(), cliente.getTelefone());
		this.cpf = cpf;
		this.identidade = identidade;
		this.tipoIdentidade = tipoIdentidade;
	}

	public String getCpf() {
		return cpf;
	}

	public String getIdentidade() {
		return identidade;
	}

	public String getTipoIdentidade() {
		return tipoIdentidade;
	}

	@Override
	public String toString() {
		return "[Pessoa Física] " + super.toString() + " CPF: " + cpf + ", Identidade: " + identidade + ", Tipo de identidade: " + tipoIdentidade;
	}
	
	
}
